### DISCUSSION TITLE
Critical dependency: the request of a dependency is an expression

### DISCUSSION THREAD

**Discussion by TLovers** [PrismJS/prism#3026] [General]
当我在nuxt 中使用prismjs 启动项目的时候命令行总是报警告 Critical dependency: the request of a dependency is an expression 。页面会报错 Cannot find module './prism-markup'
我查看prismjs/component/index.js 好像是 45行 require(pathToLanguage)的问题
当我尝试require(${pathToLanguage}) 就变得正常
我想知道这个是不是Bug

**Comment by TLovers** [comment]
Critical dependency: The request of a dependency is an expression.The command line always warns me when I start a project using Prismjs in nuxt.The page will report an error,and cannot find module './prism-markup'
The problem seems like line 45th require(pathToLanguage) when I was checking
meanwhile when I try to the require(`${pathToLanguage}`),it becomes normal
I want to know whether it is a bug

**Reply by RunDevelopment** [reply]
Is this the same issue as #3023?
